wp-content/themes/citylimits/less/series-landing-special-projects.less

```diff
@@ -280,4 +280,97 @@
       padding: 0;
     }
   }
+
+  .special-projects-flex-container-title {
```

Comment:
needs `font-family: @font-headline;`

Reply:
Whoops. I thought I did that. Updated in https://github.com/INN/umbrella-citylimits/pull/120/commits/468810004f9ec9a646f85f3ee3f32f28485b4619